.github/dependabot.yml

```diff
@@ -9,6 +9,8 @@ updates:
       - "/site"
     schedule:
       interval: "weekly"
+    allow:
+      - dependency-type: "indirect"
```

Comment:
Shouldn’t we set all to also bump the direct dependencies? See https://docs.github.com/en/code-security/dependabot/working-with-dependabot/dependabot-options-reference#dependency-type-allow.

Reply:
Or maybe:

```yaml
allow:
    - dependency-type: "direct"
    - dependency-type: "indirect"
```